hgeometry/src/Algorithms/Geometry/DelaunayTriangulation/DivideAndConquer.hs

```diff
@@ -42,6 +46,7 @@ import qualified Data.Vector as V
 --
 -- Rotating Right <-> rotate clockwise
 
+-- FIXME: rename 'delaunayTriangulation' to 'triangulate'?
```

Comment:
I think prefer 'delaunayTriangulation' over 'triangulate'; i.e. 'delaunayTriangulation pts' *is* a DT after all. We are also using something like 'intersections segs' in the line-segment intersection algorithm rather than 'computeIntersections' or so.